### DISCUSSION TITLE
Bundle OpenAM 14.7.4 with OpenDJ 4.6.5 instead of OpenDJ4.5.9

### DISCUSSION THREAD

**Discussion by pdaterao** [OpenIdentityPlatform/OpenAM#779] [Q&A]
Are there any gotchas or compatibility issues with bundling OpenAM 14.7.4 with OpenDJ 4.6.5 instead of v4.5.9?
I am not in a position to pick up OpenAM 15.0.3 yet but would like to pick up some fixes that went into OpenDJ v4.6.5. I have a build with this configuration and it is working as expected. Is there anything I should be worried about in terms of future upgrades of either OpenAM or OpenDJ? Thanks in advance.

**Comment by vharseko** [comment]
> Are there any gotchas or compatibility issues with bundling OpenAM 14.7.4 with OpenDJ 4.6.5 instead of v4.5.9?
https://github.com/OpenIdentityPlatform/OpenDJ/issues/354
